_config.yml

```diff
@@ -581,6 +581,10 @@ vendors:
   # https://github.com/ethantw/Han
   han:
 
+  # Internal version: 1.0.2
+  # https://github.com/HubSpot/pace/
+  pace: https://cdn.bootcss.com/pace/1.0.2/pace.min.js
```

Comment:
No) need this links in comments. I explain:
With the empty values (`pace:`) Next will use Internal versions of that scripts from lib directory.
With CDN values (pace: https://cdn.bootcss.com/pace/1.0.2/pace.min.js) Next will use that direct links path instead of Internal scripts in lib directory.
So, in default we always use Internal scripts, but if user want, he may use CDN links. But, he don't know where this links, and comment this links will be fine.

I will edit your pull now and u will see, how need to make rightly for ideal use any plugins like that. Do not edit anymore.